### PR TITLE
Html review

### PR DESCRIPTION
**Pull request by funilrys**
All /html file are there exceptmap.html which is a strange html file.

+ Need to review login page ==> Gonna be donne asap

### CODE REVIEW

**Comment by atais** [comment]
sorry can you summarize what is that about?
what have you changed?

the code amount is impossible to go through it :P

**Comment by funilrys** [comment]
I splited the HTML using nunjucks + can you just review the gulpfile ?

**Comment by atais** [comment]
- [ ] there is test.html in `html` files, I guess can be deleted?

I thought I have pushed mine branch - but I did just now (html-templates) that you can compare.

**Comment by funilrys** [comment]
You can safely delete it ... I used it to test something --"
